Index: header.ezt
===================================================================
RCS file: /cvsroot/viewcvs/viewcvs/templates/include/header.ezt,v
retrieving revision 1.8
diff -u -r1.8 header.ezt
--- header.ezt	27 Sep 2004 21:01:06 -0000	1.8
+++ header.ezt	19 Nov 2004 01:54:02 -0000
@@ -1,17 +1,11 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN"
- "http://www.w3.org/TR/REC-html40/loose.dtd">
-<!-- ViewCVS       - http://viewcvs.sourceforge.net/
-     by Greg Stein - mailto:gstein@lyra.org  -->
-
-<html>
-<head>
+[include "customheader.ezt"]
+  <link rel="stylesheet" href="[docroot]/styles.css" type="text/css">
   <title>[page_title]</title>
   <meta name="generator" content="ViewCVS [vsn]">
-  <link rel="stylesheet" href="[docroot]/styles.css" type="text/css">
 </head>
 
 <body>
-
+[include "menu.ezt"]
 <div class="vc_navheader">
 <table width="100%" border="0" cellpadding="0" cellspacing="0">
 <tr>
