--- post_bug.cgi.orig	2006-04-25 13:25:07.000000000 +0200
+++ post_bug.cgi	2006-09-06 02:23:26.000000000 +0200
@@ -229,6 +229,11 @@
 
 my %ccids;
 
+my $ccid = DBNameToIdAndCheck('gd-chatter@gwydiondylan.org');
+if ($ccid && !$ccids{$ccid}) {
+  $ccids{$ccid} = 1;
+}
+
 # Create the ccid hash for inserting into the db
 # use a hash rather than a list to avoid adding users twice
 if (defined $cgi->param('cc')) {
