--- /root/post_bug.cgi  Fri Sep 24 20:23:21 2004
+++ post_bug.cgi        Fri Sep 24 20:21:04 2004
@@ -207,6 +207,7 @@
 # and the list for passing to processmail
 # use a hash rather than a list to avoid adding users twice
 if (defined $::FORM{'cc'}) {
+    $::FORM{'cc'} .= ',gd-chatter@gwydiondylan.org';
     foreach my $person (split(/[ ,]/, $::FORM{'cc'})) {
         if ($person ne "") {
             my $ccid = DBNameToIdAndCheck($person);
