--- post_bug.cgi.orig	2010-08-13 11:08:40.000000000 +0200
+++ post_bug.cgi	2010-08-13 11:06:46.000000000 +0200
@@ -159,7 +159,7 @@
     $bug_params{$field} = $cgi->param($field);
 }
 $bug_params{'creation_ts'} = $timestamp;
-$bug_params{'cc'}          = [$cgi->param('cc')];
+$bug_params{'cc'}          = ['chatter@lists.opendylan.org', $cgi->param('cc')];
 $bug_params{'groups'}      = \@selected_groups;
 $bug_params{'comment'}     = $comment;
 
