Index: viewcvs.py
===================================================================
RCS file: /cvsroot/viewcvs/viewcvs/lib/viewcvs.py,v
retrieving revision 1.305
diff -u -r1.305 viewcvs.py
--- viewcvs.py	30 Oct 2004 22:12:01 -0000	1.305
+++ viewcvs.py	19 Nov 2004 01:53:40 -0000
@@ -35,7 +35,7 @@
 # development, they will remain None.
 #
 
-CONF_PATHNAME = None
+CONF_PATHNAME = r"/usr/local/viewcvs-1.0-dev/viewcvs.conf"
 
 #########################################################################
 
@@ -758,10 +758,14 @@
 # addresses.  Note that the regexps assume the text is already HTML-encoded.
 _re_rewrite_url = re.compile('((http|https|ftp|file|svn|svn\+ssh)(://[-a-zA-Z0-9%.~:_/]+)((\?|\&amp;)([-a-zA-Z0-9%.~:_]+)=([-a-zA-Z0-9%.~:_])+)*(#([-a-zA-Z0-9%.~:_]+)?)?)')
 _re_rewrite_email = re.compile('([-a-zA-Z0-9_.\+]+)@(([-a-zA-Z0-9]+\.)+[A-Za-z]{2,4})')
+_re_rewrite_bug = re.compile('([BbJj][uo][gb]):[\s]*(\d+)')
+_re_rewrite_job = re.compile('([BbJj][uo][gb]):[\s]*(\w+)')
 def htmlify(html):
   html = cgi.escape(html)
   html = re.sub(_re_rewrite_url, r'<a href="\1">\1</a>', html)
   html = re.sub(_re_rewrite_email, r'<a href="mailto:\1&#64;\2">\1&#64;\2</a>', html)
+  html = re.sub(_re_rewrite_bug, r'\1: \2 /<a href="/bugs/show_bug.cgi?id=\2">Bugzilla</a>', html)
+  html = re.sub(_re_rewrite_job, r'\1: \2 <a href="/cvszilla/job.cgi?id=\2">CVSZilla</a>', html)
   return html
 
 def format_log(log):
@@ -978,7 +982,7 @@
     # version 1.6.2.
     enscript_cmd = [os.path.normpath(os.path.join(cfg.options.enscript_path,
                                                   'enscript')),
-                    '--color', '--language=html', '--pretty-print=' + lang,
+                    '--color', '--language=html', '-E' + lang,
                     '-o', self.temp_file, '-']
     try:
       copy_stream(fp, popen.pipe_cmds([enscript_cmd]))
@@ -1113,6 +1117,7 @@
   '.cxx' : 'cpp',
   '.diff' : 'diffu',
   '.dpr' : 'delphi',
+  '.dylan' : 'dylan',
   '.el' : 'elisp',
   '.eps' : 'postscript',
   '.f' : 'fortran',
